cyclops-ui/src/components/pages/ModuleDetails.tsx

```diff
@@ -471,12 +471,15 @@ const ModuleDetails = () => {
         />
       );
     }
-
     resourceCollapses.push(
       <Collapse.Panel
         header={genExtra(resource, resource.status)}
         key={collapseKey}
-        style={{ backgroundColor: getCollapseColor(collapseKey) }}
+        style={{
+          backgroundColor: getCollapseColor(collapseKey),
+          borderTopLeftRadius: resourceCollapses.length == 0 ? "7px" : "0px",
```

Comment:
Why are you checking resourceCollapses.length == 0? Shouldn't it always be 7px? Also, why 7?

Reply:
i am checking that because 
![240517_00h17m31s_screenshot](https://github.com/cyclops-ui/cyclops/assets/119416316/25d15fb1-d017-45f8-a0c6-0a2105cfd11a)
and 7 because it fitted